ios/brave-ios/Sources/BraveUI/SwiftUI/BravePopup.swift

```diff
@@ -58,23 +58,6 @@ public struct BravePopupView<Content: View>: UIViewControllerRepresentable {
     if isPresented {
       guard uiViewController.presentedViewController == nil
       else {
-        // The system dismissed our Popup automatically, but never updated our presentation state
-        // It usually does this if you present another Popup or sheet
-        // Manually update it
-        if let controller = context.coordinator.presentedViewController
```

Comment:
Why is this code now deleted?

Reply:
Because it's not doing anything. It keeps dismissing the popup incorrectly.

I'll see if there is a different way to show a popup. It seems you're right when you dismiss on background touch, it presents again.

Reply:
Hmm I'll use the accessibility scrollView then and size it to fit. I was hoping to just keep it a decent size as the other sizes are ridiculous tbh.

Reply:
Fixed.